### ISSUE TITLE
Carthage update throw error on Xcode 9.1 

### ISSUE THREAD

**Issue by Senocico** [BrikerMan/BMPlayer#177]
If run 
`"carthage update --platform iOS BMPlayer"`

The following error will occur

```
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -project .../BMPlayer/BMPlayer.xcodeproj -scheme BMPlayer -configuration Release -derivedDataPath /.../Caches/org.carthage.CarthageKit/DerivedData/9.1_9B55/BMPlayer/1.0.0 -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive -archivePath ./ SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO (launched in .../Carthage/Checkouts/BMPlayer)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/lm/8y_p4q517l5338h3h_myx1h00000gn/T/carthage-xcodebuild.VpUHmv.log
```

How to solve this?
I really will appreciate any help.

PS: I've been looking for a solution but I haven't found yet...

**Comment by BrikerMan** [comment]
here is an error with carthage, I will try to fix that later. Please try to install with pod.

**Comment by Senocico** [comment]
Hi, would it be possible to let us know when the problem is solved?
Because I have an entire project that relies on the Carthage, and only this library is installed on CocoaPods, which makes my project not look so elegant.
Thanks a lot!

**Comment by BrikerMan** [comment]
I have released v1.0.1 to fix this issue, try to rebuild with Carthage.

**Comment by Senocico** [comment]
Thanks for quick replies! It's working flawlessly.
